packages/client/src/contents/types/seoFiles.types.ts

```diff
@@ -0,0 +1,40 @@
+import type { Config } from '../../types/index.js';
+
+export type GetSEOFiles = (
+  query: GetSEOFilesQuery,
+  config?: Config,
+) => Promise<SEOFiles>;
```

Comment:
Looking at the spec, this is not what the endpoint returns. Please, check the output and use the correct type (Tip: You can use the `PagedResponse` type to type the output)

Reply:
Resolved! ✅